build.gradle

```diff
@@ -33,25 +33,23 @@ spotless {
 
 ext {
 	dependencyVersions = [
-		apacheHttpclient:'4.5.13',
-		awsJavaSdkDynamodb:'1.12.389',
-		awsJavaSdkLambda:'1.12.290',
-		awsJavaSdkSqs:'1.12.197',
-		awsJavcSdkKms:'1.12.395',
-		awsLambdaJavaCore:'1.2.1',
-		awsLambdaJavaEvents:'3.11.0',
-		awsLambdaJavaLog4j2:'1.5.1',
-		dynamodbEnhanced:'2.17.89',
-		gson:'2.8.9',
-		jacksonDataformatYaml:'2.13.2',
-		kms:'2.17.100',
-		log4j:'2.17.1',
-		lombok:'1.18.28',
+		awsJavaSdkDynamodb:'1.12.665',
+		awsJavaSdkLambda:'1.12.665',
+		awsJavaSdkSqs:'1.12.665',
+		awsJavaSdkKms:'1.12.665',
+		awsLambdaJavaCore:'1.2.3',
+		awsLambdaJavaEvents:'3.11.4',
+		dynamodbEnhanced:'2.24.9',
+		gson:'2.10.1',
+		jacksonDataformatYaml:'2.15.3',
+		kms:'2.24.9',
+		log4j:'2.23.0',
+		lombok:'1.18.30',
 		nimbusdsOauth2OidcSdk:'9.27',
```

Comment:
This has quite a few upgrades (including some security upgrades) to pull in, but it changes the way it does JSON parsing, requiring a rewrite of lots of JSON handling code. Instead we should wait for PYIC-3908 to upgrade it.

(cc @MikeCollingwood )